lib/ansible/plugins/httpapi/checkpoint.py

```diff
@@ -14,6 +14,13 @@
   - This HttpApi plugin provides methods to connect to Checkpoint
     devices over a HTTP(S)-based api.
 version_added: "2.8"
+options:
+  domain:
+    type: str
+    description:
```

Comment:
Isn't this change already via https://github.com/ansible/ansible/pull/63976

Reply:
yeah that's a mistake, but anyway the errors seem to be from another cause

Reply:
```suggestion
    version_added: "2.10"
    description:
```